rules/github_rules/github_organization_app_integration_installed.yml

```diff
@@ -0,0 +1,47 @@
+AnalysisType: rule
+Description: An application integration was installed to your organization's Github account by someone in your organization.
+DisplayName: Github Organization App Integration Installed
+Enabled: false
+Filename: github_organization_app_integration_installed.py
+Reference: https://docs.github.com/en/enterprise-server@3.4/developers/apps/managing-github-apps/installing-github-apps
+Runbook: Confirm that the app integration installation was a desired behavior.
+Severity: Low
+Tags:
+    - Application Installation
+    - Github
+Tests:
+    - ExpectedResult: true
```

Comment:
we should provide a `ExpectedResult: false` test case to demonstrate that there are cases where the alert will not be raised.

Reply:
ack; added this test case